# Axon App message from i-Building.
float32 cmd_vel_v
float32 cmd_vel_w
uint32 set_auto_dock
uint32 ms_stop_flag
uint32 now_auto_dock_flag
uint32 left_motor_ec
uint32 right_motor_ec
